cli/commands/start.go

```diff
@@ -57,6 +57,8 @@ func init() {
 func runStartCmd(cmd *cobra.Command, args []string) error {
 	var err error
 
+	ctx.Running.StateboardCliSet = cmd.Flags().Changed("stateboard")
```

Comment:
`StateboardCliSet` looks like "a set of stateboard clis", I suggest `StateboardFlagIsSpecified` or `StateboardFlagIsSet`.